miniti-client/src/app/app.module.ts

```diff
@@ -1,18 +1,26 @@
-import { BrowserModule } from '@angular/platform-browser';
-import { NgModule } from '@angular/core';
+import {BrowserModule} from '@angular/platform-browser';
+import {NgModule} from '@angular/core';
+import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
+
+import {AppRoutingModule} from './app-routing.module';
+import {AppComponent} from './app.component';
+import {TicketListComponent} from './ticket-list/ticket-list.component';
+import {TicketEntryComponent} from './ticket-entry/ticket-entry.component';
```

Comment:
Maybe we should discuss a module structure for future stories to avoid dump things into the app.module

Reply:
This is something I am very inexperienced with so I'd love your input!

Reply:
So I had a short read through https://angular.io/guide/feature-modules and I think that would be a good idea, but the prototype seems a bit too early for this kind of complexity. I also cannot think of anything that is currently cleverly wrappable - in my head we would end up with a `TicketModule` which then includes everything. And that doesn't makes sense. :)

Reply:
Yeah, for now I'm Ok with this approach. but I would like to have a talk about the general screens we are going to have anyway.